backend/app/tasks/flow_tasks.py

```
@@ -0,0 +1,1361 @@
+            }
+
+        except SoftTimeLimitExceeded:
+            logger.error(
+                f"[flow_tasks] Execution timeout for flow {flow_id}, execution {execution_id}"
+            )
+            _handle_timeout_failure(db, execution_id, timeout_seconds, trigger_type)
+            raise
+
+        except Exception as e:
+            logger.error(
+                f"[flow_tasks] Error executing flow {flow_id}: {str(e)}",
```

Ruff (0.14.11)

Use explicit conversion flag

Replace with conversion flag

(RUF010)